td.vue/src/i18n/ar.js

```diff
@@ -47,7 +47,12 @@ const ara = {
         }
     },
     demo: {
-        select: 'حدد نموذج تهديد تجريبي من القائمة أدناه'
+        select: 'حدد نموذج تهديد تجريبي من القائمة أدناه',
+        "New Threat Model": "نموذج تهديد جديد",
```

Comment:
the names are not ideal, such as "New Treat Model", and should be renamed to something more in keeping with the rest of the attribute naming

Reply:
> the names are not ideal, such as "New Threat Model", and should be renamed to something more in keeping with the rest of the attribute naming

Actually I gave such names because , these names i am getting in the front end page (SelectDemoModel.vue) as {{model.name}} from the backend 
if you want i can change through the backend or you can suggest any other better solution ??